### PR TITLE
Milestone 3 Add Events

### PR DESCRIPTION
**Pull request by miftah1991**
Hello,

In this milestone, the below features have been added.
- Add the files calculate.js and operate.js into a logic/ directory.
- Implement the event handlers clickHandler() function.
- Use class-based components.

- No `linters `error.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,13 @@
 import { Component } from 'react';
-import Calculator from './Calculator';
+import Calculator from './components/Calculator';
```
Comment (rloterh) [inline src/App.js:3]: - [ ] Your calculator app is not functioning as intended. Running the operation 9/0 yields a blank screen as below. Kindly ensure to improve upon this aspect.

**Operation Error:**
image

File: src/App.js [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
 
 // eslint-disable-next-line react/prefer-stateless-function
```
Comment (rloterh) [inline src/App.js]: - [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Miftahuddin, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- Descriptive Pull Request
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Miftahuddin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by miftah1991** [comment]
@mikethreels 

Thank you so much

**Pull request merged**